app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Using size in dimens naming is not a good practice. We should use explanatory names for them such as `small_button_text_size`, `body_text_margin`

Reply:
Yes, I agree.